api/proto/banyandb/common/v2/common.proto

```diff
@@ -34,4 +34,5 @@ message Metadata {
     string group = 1;
     // name of the entity
     string name = 2;
+    uint32 id = 3;
```

Comment:
@lujiajing1126 the id will be generated by metadata module. We set it for index rules manually for now.